modules/ob-release-notes-1-2.adoc

```diff
@@ -5,20 +5,69 @@
 [id="ob-release-notes-1-2_{context}"]
 = Release notes for {builds-shortname} General Availability 1.2
 
-{builds-shortname} General Availability (GA) 1.2 is now available on {ocp-product-title} <version>.
+{builds-shortname} General Availability (GA) 1.2 is now available on {ocp-product-title} 4.17.
```

Comment:
Shouldn't we indicate this is available on all supported versions of OpenShift?

```suggestion
{builds-shortname} General Availability (GA) 1.2 is now available on {ocp-product-title} 4.12-4.17.
```

Reply:
That would be definitely better :+1: I will add that change.

Reply:
I will unify it with other outer loop by saying "4.12 and later versions"